pkg/networkservice/chains/nsmgr/server_test.go

```diff
@@ -44,15 +44,16 @@ import (
 	"github.com/networkservicemesh/sdk/pkg/networkservice/common/mechanisms/kernel"
 	"github.com/networkservicemesh/sdk/pkg/networkservice/core/chain"
 	"github.com/networkservicemesh/sdk/pkg/networkservice/core/next"
+	"github.com/networkservicemesh/sdk/pkg/tools/opentracing"
 	"github.com/networkservicemesh/sdk/pkg/tools/sandbox"
-	"github.com/networkservicemesh/sdk/pkg/tools/spanhelper"
 )
 
 func TestNSMGR_RemoteUsecase_Parallel(t *testing.T) {
 	defer goleak.VerifyNone(t, goleak.IgnoreCurrent())
-	logrus.SetOutput(ioutil.Discard)
 	ctx, cancel := context.WithTimeout(context.Background(), time.Second*5)
 	defer cancel()
+	ctx = logger.WithLog(ctx)
```

Comment:
Should we put a logger inside the sandbox to reduce changes?

Reply:
Yes, we can, but in this case we also need to create a logger inside sandbox/utils/NewEndpoint. 
You can take a look, PR is updated.